src/index.html

```diff
@@ -11,12 +11,98 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          class="logo"
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav>
```

Comment:
Missing empty line:

Add an empty line between sibling blocks.
There should not be a line between parent and children, but between two children of the same block only.
Add all missing empty lines from here onwards.